.buildkite/llvm-apple-pipeline.yml

```diff
@@ -1,3 +1,60 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
+
+env:
+  SETUP_GVM_VERSION: "v0.5.1"
+  IMAGE_UBUNTU_X86_64: "family/core-ubuntu-2204"
+  IMAGE_UBUNTU_ARM_64: "core-ubuntu-2004-aarch64"
+  DOCKER_REGISTRY: "docker.elastic.co"
+  STAGING_IMAGE: "${DOCKER_REGISTRY}/observability-ci"
+  DOCKER_REGISTRY_SECRET_PATH: "secret/observability-team/ci/docker-registry/prod"
```

Comment:
Is this secret available for this repository ? 

There is a secret in the ingest shared vault path:
https://github.com/elastic/package-registry/blob/d633cc4d521180f244733a8905b531eed6f75dac/.buildkite/hooks/pre-command#L12